lib/matplotlib/rcsetup.py

```diff
@@ -93,6 +94,21 @@ def __call__(self, s):
         raise ValueError(msg)
 
 
+def _single_string_color_list(s, scalar_validator):
+    try:
+        colors = mpl.color_sequences[s]
+    except KeyError as ke:
+        try:
+            # Sometimes, a list of colors might be a single string
+            # of single-letter colornames. So give that a shot.
+            colors = [scalar_validator(v.strip()) for v in s if v.strip()]
+        except ValueError as ve:
+            ke_msg = str(ke)[1:-1]  # strip off quote marks
+            raise ValueError(f'{ke_msg} and {ve}')
```

Comment:
I appreciate feedback on a better way to do the exception handling here.  This works to combine the error messages from the two options (see new test) but I feel like I'm missing something.

Reply:
Ah, so my problem was trying to be too clever 😆